sdk/storage/Azure.Storage.Files.Shares/src/ShareExtensions.cs

```diff
@@ -38,5 +38,41 @@ internal static string ToFileDateTimeString(this DateTimeOffset? dateTimeOffset)
         private static string ToFileDateTimeString(this DateTimeOffset dateTimeOffset)
             => dateTimeOffset.UtcDateTime.ToString(Constants.File.FileTimeFormat, CultureInfo.InvariantCulture);
 
+        internal static string EscapePath(this string path)
```

Comment:
I think `DataLakeExtensions` has an identical method to this.  Consider moving to Common.

Reply:
Fixed in next iteration.